test/e2e-common.sh

```diff
@@ -238,6 +261,7 @@ function knative_teardown() {
   else
     echo ">> Uninstalling Knative serving"
     echo "Istio YAML: ${INSTALL_ISTIO_YAML}"
+    echo "Cert-Manager CRD YAML: ${INSTALL_CERT_MANAGER_CRD_YAML}"
```

Comment:
CRD yaml?

Reply:
changed to Cert-Manager YAML as we changed to install full cert-manager.